pages/task/read/views/activity-log.jsx

```diff
@@ -17,36 +31,84 @@ const getStatusBadge = eventName => {
   return <span className={ className }><Snippet>{ `status.${status}.state` }</Snippet></span>;
 };
 
-const getProfile = (profile, establishmentId) => {
-  const name = getName(profile);
-  return profile.asruUser
-    ? name
-    : <Link page="profile.view" profileId={profile.id} establishmentId={establishmentId} label={name} />;
-};
+const getAuthor = (profile, task) => {
+  const name = `${profile.firstName} ${profile.lastName}`;
+  const role = getRole(profile, task);
 
-const ActivityLog = ({ task, ExtraMeta }) => {
-  if (!task.activityLog) {
-    return null;
-  }
+  return (
+    <p>
+      { role && <span className="role">{role}: </span> }
+      {
+        profile.asruUser
+          ? name
+          : <Link page="profile.view" profileId={profile.id} establishmentId={task.data.establishmentId} label={name} />
+      }
+    </p>
+  );
+};
 
+const LogItem = ({ log, task, ExtraMeta }) => {
   return (
-    <Fragment>
-      <h2><Snippet>sticky-nav.activity</Snippet></h2>
-      <ul className="task-activity">
-        { task.activityLog.map(log => (
-          <li key={log.id}>
-            {getStatusBadge(log.eventName)}
-            <p>{getProfile(log.changedBy, task.data.establishmentId)}</p>
-            <p className="comment">{log.comment}</p>
-            {
-              ExtraMeta && <p><ExtraMeta item={log} task={task} /></p>
-            }
-            <p>{format(log.createdAt, dateFormat.datetime)}</p>
-          </li>
-        ))}
-      </ul>
-    </Fragment>
+    <li key={log.id}>
+      <span className="date">{format(log.createdAt, dateFormat.datetime)}</span>
+      {getStatusBadge(log.eventName)}
+      {getAuthor(log.changedBy, task)}
+      <p className="comment">{log.comment}</p>
```

Comment:
Out of scope here as I know this is just reformatting, but this should be `<Markdown/>` and not `<p>`

Reply:
I don't think we use rich text editors for comments yet do we?